ci_scripts/ubuntu-19_10/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:19.10
+
+ENV BUILDER_NAME="ubuntu-19_10"
+
+# Update system
+RUN true \
+    && apt update \
+    && apt -y upgrade
+
+# Install generic dependencies
+RUN apt -y install curl wget git
```

Comment:
The electron-builder-*.json files indicate the packages depend on libssl and libsqlite, yet no such packages or -dev packages are installed during the building in the docker file.  That seems a little suspicious to me, but I suppose it works so I'm probably missing something.

Reply:
the electron-builder*.json files are wrong than ^^ Thought i modified them, maybe something went wrong. Thanks!